src/twisted/internet/test/test_inlinecb.py

```diff
@@ -146,18 +148,43 @@ def calling():
         d = calling()
         f = self.failureResultOf(d)
         tb = f.getTraceback()
-        self.assertIn("in erroring", tb)
+        self.assertIn("erroring", tb)
         self.assertIn("in calling", tb)
         self.assertIn("Error Marker", tb)
 
-    def test_forwardLotsOfTracebacks(self):
+    @skipIf(sys.version_info < (3, 13), "new in Python 3.13")
+    def test_forwardTracebacks_313(self):
+        """
+        Chained inlineCallbacks are forwarding the traceback information
+        from generator to generator.
+
+        A first simple test with a couple of inline callbacks for Python 3.13 and later.
+        """
+
+        @inlineCallbacks
+        def erroring():
+            yield "forcing generator"
+            raise Exception("Error Marker")
+
+        @inlineCallbacks
+        def calling():
+            yield erroring()
+
+        d = calling()
+        f = self.failureResultOf(d)
+        tb = f.getTraceback()
+        self.assertIn("yield erroring", tb)
+        self.assertIn("in calling", tb)
+        self.assertIn("Error Marker", tb)
+
+    @skipIf(sys.version_info > (3, 12), "applies to Python 3.12 and older")
+    def test_forwardLotsOfTracebacks_312(self):
         """
         Several Chained inlineCallbacks gives information about all generators.
 
-        A wider test with a 4 chained inline callbacks.
+        A wider test with 4 chained inline callbacks.
 
-        Application stack-trace should be reported, and implementation details
-        like "throwExceptionIntoGenerator" symbols are omitted from the stack.
+        Application stack-trace should not be reported.
```

Comment:
Are these changes needed?

Reply:
No, this is a hold-over from the 3.12 and 3.13 version of the test.